cms/utils/page.py

```diff
@@ -152,8 +153,10 @@ def get_page_from_request(request, use_path=None, clean_path=None):
 
     draft = use_draft(request)
     preview = 'preview' in request.GET
-    path = request.path_info if use_path is None else use_path
-
+    if request.environ.get('SCRIPT_NAME'):
```

Comment:
Instead of manually looking for `SCRIPT_NAME` i think that the easy way of achieving the same thing is by using `path` instead of `path_info`.
https://docs.djangoproject.com/en/2.2/ref/request-response/#django.http.HttpRequest.path

Reply:
@vstoykov thank you, it's much better.